### ISSUE TITLE
Support question on Readme

### ISSUE THREAD

**Issue by gkatsanos** [puma/puma-dev#208]
You mention:
`You can either build from source or download a binary from https://github.com/puma/puma-dev/releases`

What exactly does this mean? I tried cloning the repo and running "make" , "make install" but none of these work. Do you have some concrete steps you can add or paste here (I can make the PR) in order to build puma-dev for linux?

Thanks

**Comment by nonrational** [comment]
`make` will drop a `puma-dev` binary in the root of the repository. Here are the results of me running make on Ubuntu (not virtualized).

The linux version of the binary doesn't provide `-install` or `-setup` flags, so running the service in the background on Linux is not exactly "zeroconfig." 

```
[puma-dev](master)$ uname -a
Linux hostname 3.13.0-170-generic #220-Ubuntu SMP Thu May 9 12:40:49 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

[puma-dev](master)$ git show
commit 338eaccfb43c199fb7e93fb2d62a5277fcf7a7dd
Merge: 70121c7 d6656e0
Author: Nate Berkopec <nate.berkopec@gmail.com>
Date:   Wed Aug 21 11:11:31 2019 +0000

    Merge pull request #170 from matthewrudy/testing

    Add Travis and Fix the tests

[puma-dev](master)$ git status
On branch master
Your branch is up-to-date with 'origin/master'.

nothing to commit, working directory clean

[puma-dev](master)$ make
go build ./cmd/puma-dev

[puma-dev](master)$ ./puma-dev -V
Version: devel (go1.11.2)
```

**Comment by nonrational** [comment]
@gkatsanos i took a crack at making install / setup documentation more explicit here. https://github.com/puma/puma-dev/pull/209

I also added a `make install` command that will, as one might expect, place the binary in `$GOPATH`.

**Comment by nateberkopec** [comment]
Closed in #209
